Chess-FromZeroToHero.Contracts/Dtos/User/UserWithIdDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Chess_FromZeroToHero.Contracts.Dtos.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chess_FromZeroToHero.Contracts.Dtos.User
+{
+    public class UserWithIdDto : BaseWithIdDto
+    {
+        public string Username { get; set; }
+
+        public string Password { get; set; }
```

Comment:
Along the password you should also use and store a hash salt for added security.

Reply:
Yes, let's add it later